.eslintrc.json

```diff
@@ -0,0 +1,281 @@
+{
+  "root": true,
```

Comment:
It looks like this file is the jsdom eslintrc?
https://github.com/jsdom/jsdom/blob/master/.eslintrc.json

Can we specify that file as a parent to this one instead of needing to copy it?